### PR TITLE
Updated the Design System section

### PR DESCRIPTION
**Pull request by davidc-gds**
- Changed the heading to active language instead of passive
- Added a sentence about and link to the GOV.UK Prototyping Kit

### CODE REVIEW

**Review by DavidBiddle — DISMISSED** [review]

File: source/manuals/accessibility.html.md.erb [hunk 1]
```diff
@@ -79,12 +79,14 @@ You can test your service using an [accessibility empathy lab](https://gds.blog.
...
 Testing with real users can be a time consuming process. It is recommended that this is done only after other accessibility testing takes place.
 
-### Consider using the GOV.UK Design System
+### Use the GOV.UK Design System
 
 The [GOV.UK Design System](https://design-system.service.gov.uk/) is a collection of website styles, components and patterns designed to be used to build government services. It provides pre-built website markup that has been developed specifically to be accessible. Using it will also save development time and provide a look and feel consistent with other government services.
 
 While it may not have a full set of components for every service, it provides a solid foundation to work from that can be expanded to suit a service's individual needs.
 
+The GOV.UK Design System also includes the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/), which can be used to show ideas to people you work with and to do user research.
```
Comment (joelanman) [inline source/manuals/accessibility.html.md.erb:88]: nice updates! I think I'd drop the 'includes' part, something like

You can also use the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/), to quickly create working prototypes to share ideas with people and to do user research.

Comment (davidc-gds) [reply source/manuals/accessibility.html.md.erb]: Thanks @joelanman - that seems sensible to me 👍
I've dropped the 'includes' part and tightened up the content a bit. See if this looks good to you:

> You can also use the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/) to quickly create working prototypes, share ideas with people and conduct user research.

**Review by joelanman — APPROVED** [review]
Looks good to me, thanks!

File: source/manuals/accessibility.html.md.erb [hunk 2]
```diff
@@ -79,12 +79,14 @@ You can test your service using an [accessibility empathy lab](https://gds.blog.
...
 Testing with real users can be a time consuming process. It is recommended that this is done only after other accessibility testing takes place.
 
-### Consider using the GOV.UK Design System
+### Use the GOV.UK Design System
 
 The [GOV.UK Design System](https://design-system.service.gov.uk/) is a collection of website styles, components and patterns designed to be used to build government services. It provides pre-built website markup that has been developed specifically to be accessible. Using it will also save development time and provide a look and feel consistent with other government services.
 
 While it may not have a full set of components for every service, it provides a solid foundation to work from that can be expanded to suit a service's individual needs.
 
+You can also use the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/) to quickly create working prototypes, share ideas with people and conduct user research.
```
Comment (NoraGDS) [inline source/manuals/accessibility.html.md.erb:88]:
```suggestion
You can also use the [GOV.UK Prototype Kit](https://govuk-prototype-kit.herokuapp.com/) to create working prototypes, share ideas with people and conduct user research.
```

Comment (NoraGDS) [reply source/manuals/accessibility.html.md.erb:88]: Suggest removing 'quickly'.

Comment (lfdebrux) [reply source/manuals/accessibility.html.md.erb:88]: 😅

**Comment by davidc-gds** [comment]
Marking this change as `ready-to-merge-final-final-3` if one of the reviewers wants to take a look and then merge 🤞

I don't personally have permission to merge into GDS Way.

**Review by lfdebrux — APPROVED** [review]

**Pull request merged**